#goal definition
# 1:hand brake release, 2:hand brake engage
uint8 operation
---
#result definition
bool success
---
#feedback
bool busy

